test/controllers/PaypalControllerSpec.scala

```diff
@@ -223,7 +228,7 @@ class PaypalControllerSpec extends PlaySpec
       val result: Future[Result] = Helpers.call(fixture.controller.capturePayment, captureRequest)
       status(result).mustBe(200)
 
-      fixture.numberOfCallsToStoreMetaDataMustBe(1)
+      Eventually.eventually(timeout(Span(5, Seconds)), interval(Span(50, Millis))) {fixture.numberOfCallsToStoreMetaDataMustBe(1)}
```

Comment:
Do we need to wrap this in an eventually too? I'd have thought that using the eventually in numberOfCallsToStoreMetaDataMustBe would be sufficient.

Reply:
good catch!